src/Tools/dotnet-trace/Profile.cs

```diff
@@ -22,7 +22,9 @@ public Profile(string name, IEnumerable<EventPipeProvider> providers, string des
 
         public string Description { get; }
 
-        public bool Rundown { get; set; } = true;
+        public long RundownKeyword { get; set; }
+
+        public RetryStrategy RetryStrategy { get; set; } = RetryStrategy.DropKeywordKeepRundown;
```

Comment:
The default values for `Profile.RetryStrategy` and `MonitoringSourceConfiguration.Strategy` are different. I would expect them to be the same.